content/en/getting_started/agent.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Getting Started with the Agent
+kind: documentation
+further_reading:
+- link: "/agent/basic_agent_usage/"
+  tag: "Documentation"
+  text: "Basic Agent Usage"
+- link: "/agent/faq/why-should-i-install-the-agent-on-my-cloud-instances/"
+  tag: "FAQ"
+  text: "Why should I install the Datadog Agent on my cloud instances?"
+---
+
+## Overview
+The Agent is lightweight software installed on your hosts. It reports metrics and events from your host to Datadog. With additional setup, the Agent can report [live processes][1], [logs][2], and [traces][3].
+
+## Setup
+If you haven't already, create a [Datadog account][4].
+
+### Installation
+The Agent can be installed on many different platforms. Most systems have a one-line install option.
+
+{{< partial name="platforms/platforms.html" desc="Choose your platform to see installation instructions:" links="gs" >}}
+
+### Configuration
+The Agent's [main configuration file][5] is `datadog.yaml`. The only required parameter is your [Datadog API key][6] which is used to associate your Agent's data with your organization. See the [sample config_template.yaml][7] for all available configuration options.
+
+### Validation
+Run the Agent's [status command][8] to verify installation.
+
+## Data Collected
+
+### Metrics
+By default, the Agent collects metrics with the following checks. To collect metrics from other technologies, see the [Integrations][9] page.
+
+* [Disk][10]
+* [Network][11]
+* [NTP][12]
+* [System][13]
+
+### Events
+The Agent sends events to Datadog when an Agent is started or restarted.
```

Comment:
This should link out to the start, restart and other Agent commands

Reply:
It's loosely related and doesn't feel like it fits here. I will add it elsewhere.